src/views/Accessibility/index.tsx

```diff
@@ -57,7 +57,13 @@ const Accessibility = () => {
   const RequestDetails = (
     <SecureRoute
       path="/508/requests/:accessibilityRequestId"
-      component={AccessibilityRequestDetailPage}
+      render={() => {
+        return (
+          <AccessibilityRequestDetailPage
+            isAccessibilityTeam={user.isAccessibilityTeam(userGroups, flags)}
```

Comment:
This feels like the wrong spot for this, especially since the request details page itself can access this information. I think we need to move this check in to the request page.